versioned_docs/version-2.7/integrations-in-rancher/cloud-marketplace/aws-marketplace-payg-integration/troubleshooting.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Troubleshooting Rancher Prime PAYG Cluster in AWS
+---
+
+This section contains information to help troubleshoot issues when installing the Rancher Prime PAYG offering.
+
+## Jobs and Pods
+
+Check the status of pods or jobs:
+
+```shell
+kubectl get pods --all-namespaces
+```
+
+If a pod is not in a `Running` state, you can attempt to find the root cause with the following commands:
+
+- Describe pod: `kubectl describe pod $POD_NAME -n $NAMESPACE`
+- Pod container logs: `kubectl logs $POD_NAME -n $NAMESPACE`
+- Describe job: `kubectl describe job $JOB_NAME -n $NAMESPACE`
+- Logs from the containers of pods of the job: `kubectl logs -l job-name=$JOB_NAME -n $NAMESPACE`
```

Comment:
Just a reminder about placeholders. We're really inconsistent about our placeholder syntax throughout the docs, but if we're adding new content we should stick to `<angle-brackets>`